packages/web-components/src/components/carousel/carousel.ts

```diff
@@ -111,6 +114,39 @@ class DDSCarousel extends LitElement {
     }
   }
 
+  /**
+   * Handles card focus throughout pages.
+   *
+   * @param event The event.
+   */
+  @HostListener('shadowRoot:focusin')
+  // @ts-ignore: The decorator refers to this method but TS thinks this method is not referred to
+  private _handleFocus = async ({ target, relatedTarget }: FocusEvent) => {
+    const currentContains = target !== this && this.contains(target as DDSCard);
+    const oldNotContains = target !== this && !this.contains(relatedTarget as DDSCard);
+    const currentCardIndex = Array.from(this.children).indexOf(target as HTMLElement);
+
+    // keep current page if tabbing back into the carousel after previously moving pages
+    if (currentContains && oldNotContains && currentCardIndex === 0) {
+      (this.children[this.start] as HTMLElement).focus();
+      return;
+    }
```

Comment:
Do we still need this logic?

Reply:
@asudoh Yes, we still need that logic to focus on the current starting card after having tabbed outside the carrousel. Without it, the focus will always go to the first card automatically, out of view.

Reply:
I see, thank you for clarifying - I suggest preserving the "focused card index" in `_handleFocus()`. It can be done by "old contains but current node contains" detection and preserving `event.relatedTarget`. In this way we can set `this.start` according to the index of such "previously focused card".